playlists-prod/powerpoint.yaml

```diff
@@ -94,6 +103,15 @@
     https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/powerpoint/slide-management/get-slide-metadata.yaml
   group: Slide Management
   api_set: {}
```

Comment:
BTW. do you know what api set these should be?

Reply:
They are 1.5